tests.py

```diff
@@ -100,3 +107,21 @@ def test_sip_api_qtpy():
         assert sip.getapi("QString") == 2, ("PyQt4 API version should be 2, "
                                             "instead is %s"
                                             % sip.getapi("QString"))
+
+
+def test_sip_api_already_set():
+    """Qt.py should cause ImportError when sip API v1 was already set
```

Comment:
First line of a docstring should be a short summary, with no line breaks.

Subsequent lines can explain in more detail. Also not necessary to mention Qt.py, as all tests are about Qt.py.